clang/lib/CIR/Dialect/Transforms/TargetLowering/TargetInfo.h

```diff
@@ -27,9 +27,20 @@ enum class X86AVXABILevel {
   AVX512,
 };
 
+/// The ABI kind for AArch64 targets.
+enum class AArch64ABIKind {
```

Comment:
Same as what we have in `clang/lib/CIR/Dialect/Transforms/LoweringPrepareCXXABI.h`, please unify already!